setup-container.sh

```diff
@@ -57,7 +57,7 @@ EOF
 
 DOCKER_SONIC_MGMT="docker-sonic-mgmt"
 DOCKER_REGISTRY="sonicdev-microsoft.azurecr.io:443"
-LOCAL_IMAGE_NAME=docker-sonic-mgmt-$USER
+LOCAL_IMAGE_NAME=docker-sonic-mgmt-${USER,,}
```

Comment:
This method doesn't work for me in non-Bash shells (I think maybe it's not POSIX compliant and Bash specific?). Could we change to something like this?
```suggestion
LOCAL_IMAGE_NAME=docker-sonic-mgmt-`echo "$USER" | tr '[:upper:]' '[:lower:]'`
```